tools/DynamoPackagesAnalyzer/DynamoPackagesAnalyzer.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
when built, where does this project output to?

Reply:
To the bin/ folder which is a relative path to root folder

Reply:
sorry, relative to the root of this project, or the main bin folder relative to the repo root? 
We want to keep this project OUT of the main bin folder that gets delivered to clients.

Reply:
To the root of this project, currently, I'm not importing any kind of shared configuration from Dynamo